packages/terra-application-navigation/src/ApplicationNavigation.jsx

```diff
@@ -63,6 +63,18 @@ const propTypes = {
    * `navigationItems` array.
    */
   activeNavigationItemKey: PropTypes.string,
+  /**
+    * The id for the logout utility item.
+    */
```

Comment:
nit : formatting seems to off here compare to other props.

Reply:
This prop has been removed. Marking as resolved.